docs/azmcp-commands.md

```diff
@@ -397,8 +397,14 @@ azmcp grafana list --subscription <subscription>
 ### Azure MCP Best Practices
 
 ```bash
+# Get best practices for secure, high-quality Azure Functions app code generation.
+azmcp bestpractices azurefunctions get_code_generation
```

Comment:
Isn't it get-code-generation?

Reply:
I updated the code and docs to use dash separated names.